### PR TITLE
[DS-1596][DS-1597][DS-1598]: add fixes

### PR DESCRIPTION
**Pull request by david-storm**

### CODE REVIEW

File: fullcalendar-app/src/components/FullCalendar.vue [hunk 1]
```diff
@@ -448,19 +450,19 @@ export default {
           // Determine the optimal vertical position for the popover
           let top;
           // Check if there's enough space above the event for the popover
-          if (eventRect.top - popoverRect.height >= calendarRect.top) {
+          if (eventRect.top > popoverRect.height) {
             // Position the popover above the event
-            top = eventRect.top - popoverRect.height;
+            top = eventRect.top - (popoverRect.height ) - calendarFull.top;
```
Comment (podarok) [inline fullcalendar-app/src/components/FullCalendar.vue:455]: What if there is no `calendarFull.top` ?

File: fullcalendar-app/src/components/FullCalendar.vue [hunk 2]
```diff
@@ -448,19 +450,19 @@ export default {
...
           } else if (eventRect.bottom + popoverRect.height <= calendarRect.bottom) {
             // Position the popover below the event if there's not enough space above
-            top = eventRect.bottom;
+            console.log(eventRect.bottom - calendarFull.top)
```
Comment (podarok) [inline fullcalendar-app/src/components/FullCalendar.vue]: Let's avoid debug code
```suggestion
```

File: fullcalendar-app/src/components/FullCalendar.vue [hunk 3]
```diff
@@ -504,4 +506,7 @@ export default {
 .fc-event-time {
   opacity: 80%;
 }
+.fc-day-today {
+  background-color: inherit !important;
```
Comment (podarok) [inline fullcalendar-app/src/components/FullCalendar.vue]: Why do we need `!important` here?

**Review by podarok — CHANGES_REQUESTED** [review]
See suggestion and questions

File: fullcalendar-app/src/components/FullCalendar.vue (hunk 1 above)
Comment (david-storm) [reply fullcalendar-app/src/components/FullCalendar.vue:455]: calendarFull it's main app, I think it there always

File: fullcalendar-app/src/components/FullCalendar.vue (hunk 2 above)
Comment (david-storm) [reply fullcalendar-app/src/components/FullCalendar.vue]: ok

File: fullcalendar-app/src/components/FullCalendar.vue (hunk 3 above)
Comment (david-storm) [reply fullcalendar-app/src/components/FullCalendar.vue]: We don't need this

Comment (podarok) [reply fullcalendar-app/src/components/FullCalendar.vue]: Let's remove then. Less `!important` - more predictable UI

**Pull request merged**